queue_services/entity-emailer/src/entity_emailer/email_processors/nr_notification.py

```diff
@@ -60,17 +60,23 @@ def process(email_info: dict, option) -> dict:
     expiration_date = ''
     if nr_data['expirationDate']:
         exp_date = datetime.fromisoformat(nr_data['expirationDate'])
-        expiration_date = exp_date.strftime('%Y-%m-%d')
+        expiration_date = exp_date.strftime('%B %d, %Y %H:%M %p')
 
     refund_value = ''
     if option == Option.REFUND.value:
         refund_value = email_info.get('data', {}).get('request', {}).get('refundValue', None)
 
+    for n_item in nr_data['names']:
+        if (n_item['state'] == 'APPROVED') or (n_item['state'] == 'CONDITION'):
```

Comment:
this can be changed to using **in**
```suggestion
        if n_item['state'] in ('APPROVED', 'CONDITION'):
```

Reply:
done and pushed into PR